src/web/components/AuditTrail/AuditTrail.scss

```diff
@@ -0,0 +1,73 @@
+.audit-trail-table-container {
+  .audit-log-table {
+    font-size: 0.875rem;
+    width: 100%;
+
+    thead {
+      font-weight: 700;
+      border: none;
+    }
+
+    td {
+      padding-top: 15px;
+      padding-bottom: 15px;
+
+      &.succeeded {
+        width: 5%;
+        text-align: center;
+      }
+
+      &.event-data {
+        width: 50%;
+      }
+
+      .icon_x {
+        color: red;
+      }
+
+      .icon_check {
+        color: green;
+      }
+    }
+  }
+
+  .audit-trail-table-header {
+    display: flex;
+    justify-content: end;
+    align-items: baseline;
+    padding-bottom: 6px;
```

Comment:
We could make this 10px to align with the search on the sharing permissions screen